bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewerCommand.java

```diff
@@ -82,43 +83,53 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             return;
         }
 
-        var reviewer = parseUser(matcher.group(2), pr, censusInstance);
-        if (reviewer.isEmpty()) {
-            reply.println("Could not parse `" + matcher.group(2) + "` as a valid reviewer.");
-            showHelp(pr, reply);;
-            return;
+        var reviewers = new ArrayList<Contributor>();
+        for (var entry : matcher.group(2).split(" ")) {
```

Comment:
Should we allow "," as a separator as well?

Reply:
Yeah that makes sense, let me fix :+1: